packages/connect-iframe/webpack/core.webpack.config.ts

```diff
@@ -3,8 +3,16 @@ import webpack from 'webpack';
 import merge from 'webpack-merge';
 
 import baseConfig from './base.webpack.config';
+import { getDistPathForProject } from './utils';
 
-const DIST = path.resolve(__dirname, '../build');
+const project = process.env.PROJECT || 'iframe';
+
+if (project !== 'iframe' && project !== 'suite-web' && project !== 'popup') {
+    throw new Error(`Unsupported project: ${project}`);
+}
+const DIST = getDistPathForProject(project);
+
+const CORE_PUBLIC_PATH = process.env.ASSET_PREFIX ? `${process.env.ASSET_PREFIX}/` : '/';
```

Comment:
note, this is little susceptible to not defining ASSET_PREFIX with a trailing '/'. I am just brainstorming whether it would make sense to validate it, ideally using some standard API 🤔. Somewhere I did string.replaceAll('//'. '/'), just in case. but it is not very nice